src/main/resources/i18n/messages_de.properties

```diff
@@ -793,4 +793,5 @@ settings.general.startTab = Starter "Tab"
 startTab.selectTab = Wählen Sie den "Tab", mit dem Sie beginnen möchten\:
 startTab.message = Hier können Sie den "Tab" auswählen, mit dem der Client startet. Sie können dies später in den Einstellungen ändern, wenn Sie möchten.
 startTab.save = Speichern
-vault.replays.onlyShowLastYear = Nur ein Jahr alte
\ No newline at end of file
+vault.replays.onlyShowLastYear = Nur ein Jahr alte
+chat.filter.countryPrompt=Country
```

Comment:
Translations are made on poeditor.com don't ever commit changes to message properties file other than the English one

Reply:
Roger that. I have removed the changes for the localized files.